### DISCUSSION TITLE
FYI. Another way to patch OpenAI

### DISCUSSION THREAD

**Discussion by MRYingLEE** [promplate/pyodide#3] [Ideas]
FYI. There is another way to patch, https://github.com/cloudflare/pyodide/blob/main/packages/httpx/httpx_patch.py.

I have no idea your way or this better, but for your reference.

**Comment by CNSeniorious000** [comment]
Thanks. I've read that, which is indeed a better way. I'm may use this instead in a few days. Or of course you are welcome to make a PR for this if you like.

---

The only downside is that this would need to install the `httpx` package, whose resolution may be somehow expensive (but probably that doesn't matter because the pyodide runtime is much larger, which is the real bottleneck). I suppose I may use this as an optional approach.

**Comment by CNSeniorious000** [comment]
I decided to implement that in a separate package, see https://github.com/pyodide/pyodide/discussions/4999
